src/commands/install.ts

```diff
@@ -30,43 +42,92 @@ export default class Install extends Command {
     }),
   };
 
-  private async installSingle(service_id: string) {
-    const service_dir = process.cwd();
-    const parts = service_id.split(':');
-    const service_name = parts[0];
-    let service_tag = parts[1];
-    const config = ServiceConfig.loadFromPath(service_dir);
+  async run() {
+    await this.tasks();
```

Comment:
Seems redundant now that we're not using Listr. Let's just make `tasks()` the new `run()` method